content/events/2021-online-unconference/_index.md

```diff
@@ -9,77 +9,77 @@ description = "Nordic RSE – Research Software Engineers in the nordics – Unc
 - *Demonstrations:* show us some tools or software that you like
 - *Workshop or [ReproHacks](https://reprohack.github.io/reprohack-hq/)* (only 3 hours in a day)
 - *Talks*: teach us something new
 - *On-going projects*: share with others something unfinished and almost working that you would like to get feedback on
 - *Something else*: is there something you would like to share with others that does not fit any of the previous points? We are looking forward to hearing about it!
 
+Ready to submit your idea? - Please use the [unconference contribution issue template](https://github.com/nordic-rse/nordic-rse.github.io/blob/main/.github/ISSUE_TEMPLATE/unconference-contribution.md) to let us know. It does not have to be polished yet, but so that the community gets the idea. We can polish it together if you want to :)
 
-## How to contribute ideas
+You may also come and chat with us in the [Zulip chat](https://coderefinery.zulipchat.com/#narrow/stream/213720-nordic-rse), where we can figure out contributions together.
 
-- Would like to participate but don't really know how? Chat with us in the [Zulip chat](https://coderefinery.zulipchat.com/#narrow/stream/213720-nordic-rse), we will figure out something together.
-- Once you have an idea for your contribution, open a github issue [here](https://github.com/nordic-rse/nordic-rse.github.io/issues/new/choose), use the *Unconference contribution* template.
-- You can also email us at <nordic-rse-organizers@neic.no>. In this case, please try to follow [this template](https://github.com/nordic-rse/nordic-rse.github.io/blob/main/.github/ISSUE_TEMPLATE/unconference-contribution.md)
-- Note that **you don't need to submit a long and fully polished proposal**, you can update your idea until the unconference and you will receive all the feedback and support that you need.
+Uncomfortable with github issues? - you can also email us at <nordic-rse-organizers@neic.no>. In this case, please try to follow [this template](https://github.com/nordic-rse/nordic-rse.github.io/blob/main/.github/ISSUE_TEMPLATE/unconference-contribution.md) as much as possible.
 
-- For organisation purposes, we kindly ask you to **submit your idea by June 28th 2021** (1-2 paragraphs text may be enough).
-- If you miss the deadline, you are still welcome to join the unconference. You will have the chance to add your contribution live during the event. But the earlier we know, the easier it is for us to schedule the contributions.
-- Any kind of contribution is welcomed, however please bear in mind that
-   - we will publish name and abstracts of contributions.
-   - contributions have to respect the [Code of Conduct](./code-of-conduct).
+If you want to be part of the published program, we kindly ask you to **submit your idea by June 28th 2021** via any of the above mentioned channels.
```

Comment:
Maybe we could say that the program is updated every time a new idea is submitted.

Reply:
mentioned below